region/maridia/inner-green.json

```diff
@@ -606,29 +610,37 @@
             {
               "id": 2,
               "requires": [
-                "Grapple",
-                {"or": [
-                  {"and": [
-                    "canSwim",
-                    {"or": [
-                      "HiJump",
-                      "SpaceJump",
-                      {"and": [
-                        "canIBJ",
-                        "canUseSpringBall"
-                      ]},
-                      "canGravityWalljump"
-                    ]}
-                  ]},
-                  {"and": [
-                    "canSuitlessMaridia",
-                    "SpaceJump",
-                    "HiJump",
-                    {"or": [
-                      "canSpringBallJumpMidAir",
-                      "canFlatleyTurnaroundJump"
-                    ]}
-                  ]}
+                {"or":[
+                   {"and": [
+                     "Grapple",
+                     {"or": [
+                       {"and": [
+                         "canSwim",
+                         {"or": [
+                           "HiJump",
+                           "SpaceJump",
+                           {"and": [
+                             "canIBJ",
+                             "canUseSpringBall"
+                           ]},
+                           "canGravityWalljump"
+                         ]}
+                       ]},
+                       {"and": [
+                         "canSuitlessMaridia",
+                         "SpaceJump",
+                         "HiJump",
+                         {"or": [
+                           "canSpringBallJumpMidAir",
+                           "canFlatleyTurnaroundJump"
+                         ]}
+                       ]}
+                     ]}
+                   ]},
+                   {"and": [
+                     "canXrayClimb",
```

Comment:
Because this climb starts underwater, I think it additionally needs either `canSwim` or `canSuitlessMaridia`.

Reply:
yep.  missed this one